migration/iD11/oracle/202303101208_IDEMPIERE-5622.sql

```diff
@@ -0,0 +1,14 @@
+-- IDEMPIERE-5622 - Language in Window and Info Window customization should be filtered by log in languages
+SELECT register_migration_script('202303101208_IDEMPIERE-5622.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Mar 10, 2023, 12:08:54 PM CET
+UPDATE AD_Column SET AD_Reference_Value_ID=327,Updated=TO_TIMESTAMP('2023-03-10 12:08:54','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_UU='0e32e607-dbc8-47bd-844f-9bc1fc772b39'
```

Comment:
@hengsin is that ok? Or should I manually change the AD_Column_UU to its corresponding official AD_Column_ID?

Reply:
Changed in the new commit